### PR TITLE
Added ps and blank lines format

### PR DESCRIPTION
**Pull request by WJWang**
Plz merged and relint correctly next time

### CODE REVIEW

File: doc.txt [hunk 1]
```diff
@@ -1,7 +1,11 @@
 為聲請支付命令事：
 一、請求標的
-（一）債務人應給付債權人新台幣（下同）貳萬參仟玖佰零壹元，並自本件支付命令送達之翌日起至清償日止，按週年利率百分之五計算之利息。
-（二）督促程序費用由債務人負擔。
+	（一）債務人應給付債權人新台幣（下同）貳萬參仟玖佰零壹元，並自本件支付命令送達之翌日起至清償日止，按週年利率百分之五計算之利息。
+
+	（二）督促程序費用由債務人負擔。
```
Comment (SEandec5) [inline doc.txt]: Too much spaces, i think 2 spaces is enough.

**Review by SEandec5 — COMMENTED** [review]
Others, LGTM.

File: doc.txt (hunk 1 above)
Comment (WJWang) [reply doc.txt]: OK, I will fix it.

**Review by SEandec5 — COMMENTED** [review]
Thank

**Pull request merged**